lib/rules/custom-property-pattern/__tests__/index.js

```diff
@@ -125,3 +125,18 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	customSyntax: 'postcss-scss',
+	config: ['$foo-color'],
```

Comment:
@jeddy3 any hints on relevant config here? ref to relevant documentation on possible values?

Reply:
See https://github.com/stylelint/stylelint/pull/5925#discussion_r811145249